exchanges/huobi/huobi_test.go

```diff
@@ -1356,9 +1357,8 @@ func TestWSOrderbook(t *testing.T) {
 	assert.Equal(t, 0.56281, liq, "Bid Liquidity should be correct")
 }
 
-// TestWSTradeDetail checks we can send a trade detail through
-// We can't currently easily see the result with the current DB instance, so we just check it doesn't error
-func TestWSTradeDetail(t *testing.T) {
+// TestWsHandleAllTradesMsg checks we can send a trade detail through
+func TestWsHandleAllTradesMsg(t *testing.T) {
```

Comment:
🚧  Change Request:
* Ws => WS ; See [Initialisms](https://google.github.io/styleguide/go/decisions.html#initialisms)
* Combined with suggestion above
```suggestion
// TestWSHandleAllTradesMsg ensures wsHandleAllTrades sends trade.Data to the ws.DataHandler
func TestWSHandleAllTradesMsg(t *testing.T) {
```

Reply:
updated